products/amazon-glue.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Amazon Glue
+category: service
+tags: amazon glue
+iconSlug: amazonaws
+permalink: /amazon-glue
+releasePolicyLink:
+  https://docs.aws.amazon.com/glue/latest/dg/glue-version-support-policy.html
+releaseDateColumn: true
+releaseColumn: false
+
+auto:
+-   custom: true
+
+# Versions taken from https://docs.aws.amazon.com/glue/latest/dg/release-notes.html
+releases:
+-   releaseCycle: "4.0"
+    releaseDate: 2022-11-28
+    eol: false
+    pythonVersion: "3.10"
+    sparkVersion: "3.3"
+
+-   releaseCycle: "3.0"
+    releaseDate: 2021-08-19
+    eol: false
+    pythonVersion: "3.7"
+    sparkVersion: "3.1"
+
+-   releaseCycle: "2.0"
+    releaseDate: 2020-08-10
+    eol: 2024-01-31
+    pythonVersion: "3.7"
+    sparkVersion: "2.4"
+
+---
+
+> [Amazon Glue](https://aws.amazon.com/glue/) is a serverless data integration service that makes
+> it easier to discover, prepare, move, and integrate data from multiple sources for analytics,
+> machine learning (ML), and application development.
+
+The AWS Glue version determines the runtime versions of Apache Spark and Python. New versions of
+Glue generally include major version upgrades for Apache Spark and Python, so be sure to refer to
+Spark and Python migration guides when upgrading to a newer version of Glue.
+
```

Comment:
Maybe a summary of this (seen on https://docs.aws.amazon.com/glue/latest/dg/glue-version-support-policy.html) could also be documented, WDYT ?

> Occasionally AWS Glue discontinues support for old AWS Glue versions. However, jobs running on deprecated versions are no longer eligible for technical support. AWS Glue will no longer apply security patches or other updates to deprecated versions. AWS Glue will also not honor SLAs when jobs are run on deprecated versions.

Reply:
I think that is a great call. I just added a simplified version of this paragraph to the pull request